messaging/warriors/tasks/deploy.ts

```diff
@@ -113,7 +101,7 @@ const setInteractors = async (
   }
 };
 
-const descTask = `Deploy the contract`;
-const descNetworksFlag = `Comma separated list of networks to deploy to`;
-
-task("deploy", descTask).addParam("networks", descNetworksFlag).setAction(main);
+task("deploy", "Deploy the contract", main).addParam(
+  "networks",
+  "Comma separated list of networks to deploy to"
```

Comment:
maybe can also list the options of networks here?

Reply:
Good idea! f2a5e67